docs/roadmap.md

```diff
@@ -1,53 +1,43 @@
+  - Continuing to support File Explorer enhancements that we started in the 1.4 release
+  - Adding controls to close the WinUI 2 to WinUI 3 gap for developers interested in porting their apps, including but not limited to the following:
+    - Segmented control
+    - Labels in the command bar flyout top bar items
+    - Maps control
+    - Investigations into the table view and ink controls
+  - Supporting highly functional tabbed windows that work with touch, snapping, and other OS features
+  - Drag and drop support in WebView2
+  - Improved screen reader support, text scaling support, and other accessibility features
+  - Fixes, documentation, and recommendations on how to test WinUI apps
+  - Stability and performance improvements based on the prioritized GitHub bug backlog
+- **Dynamic Lighting**
+  - Onboarding new and exciting APIs to the Windows App SDK
 
 ## Join in and stay connected!
 
 Technologies within the Windows App SDK have already benefitted greatly from the incredible feedback and insights the community has provided around features to build, and bugs to fix. 
-*Thank you* for your continued passion and drive to make Windows App SDK the best platform it can be; it has really made a difference!
 
-We’re working on new strategies to try and tackle the volume of feedback in a timely manner. 
-As we concentrate on the areas of focus for 2022 mentioned above, we’ll prioritize responding to feedback in those areas first, and we’ll also publish clearer policies on handling stale issues that are unlikely to be addressed within the next couple releases.
+*Thank you* for your continued passion and drive to make Windows App SDK the best platform it can be; it has really made a difference!
 
-You can join in by contributing to existing issues, or filing a new issue on the [Windows App SDK](https://github.com/microsoft/WindowsAppSDK) or [WinUI](https://github.com/microsoft/microsoft-ui-xaml) GitHub repos. If you're interested in WinUI 3, come join us on our live monthly [WinUI Community Calls](https://www.youtube.com/playlist?list=PLI_J2v67C23ZqsolUDaHoFkF1GKvGrttB), which are hosted on the third Wednesday of every month at 9am PT. Finally, you can stay in touch with us on Twitter using [#WindowsAppSDK](https://twitter.com/search?q=%23WindowsAppSDK). 
\ No newline at end of file
+You can join in by contributing to existing issues, or filing a new issue on the [Windows App SDK](https://github.com/microsoft/WindowsAppSDK) or [WinUI](https://github.com/microsoft/microsoft-ui-xaml) GitHub repos. If you're interested in WinUI 3, come join us on the live Windows Development [Community Calls](https://www.youtube.com/playlist?list=PLI_J2v67C23ZqsolUDaHoFkF1GKvGrttB), which are hosted quarterly. Finally, you can stay in touch with us on Twitter using [#WindowsAppSDK](https://twitter.com/search?q=%23WindowsAppSDK).
```

Comment:
>Twitter [](http://example.com/codeflow?start=457&length=7)

"X"?

Reply:
Maybe X (formerly Twitter)?

Reply:
Given it's a search tag, perhaps just delete the whole sentence since it's not a real method of "reaching us". And replace it with something like:

_Tag us on social media using `#WindowsAppSdk` too!`

Reply:
Thanks for the suggestions! I actually mostly left this intact from the last version of the roadmap page, but I agree it could be updated.